src/lib/hosts-file.ts

```diff
@@ -0,0 +1,123 @@
+import fs from 'fs';
+import { platform } from 'os';
+import { promisify } from 'util';
+import sudo from 'sudo-prompt';
```

Comment:
Should it be [`@vscode/sudo-prompt`](https://www.npmjs.com/package/@vscode/sudo-prompt)? The [former](https://www.npmjs.com/package/sudo-prompt) is reported deprecated.

Reply:
maybe, but this one is already used in Studio before this PR. So seems better addressed separately.